.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,7 +7,28 @@ assignees: ''
 
 ---
 
-**This bug is related to the**
+# Bug report
+
+You must read the information below before you submit a bug report. 
+
+The MTD API team cannot help if your bug report is about:
+
+- Fraud Prevention Headers
+- Issues relating to OAuth2
+- Subscribing or registering to use the API Platform, MTD enrolment, de-enrolment, registration for VAT, Beta services etc.
```

Comment:
Should there be a comma after “Beta Services”

Reply:
Googled it. Yes.